marshal/admin.go

```diff
@@ -303,14 +311,19 @@ func (a *consumerGroupAdmin) SetConsumerGroupPosition(groupID string,
 	defer close(claimFailures)
 
 	var claimsWg sync.WaitGroup
-	// stopHeartbeating channel instructs all successfully claimed and heartbeating claims to stop.
-	stopHeartbeating := make(chan struct{})
+	// Sending a message on stopHeartbeat instructs all successfully claimed and
```

Comment:
I would say `Closing stopHeartbeat` because sending a message would only instruct one to stop.